src/Core/Commit.js

```diff
@@ -11,17 +11,12 @@ function commitElement (element) {
 				return elementFragment(element)
 			case String:
 			case Number:
-				return elementText(element)
-			case Function:
-			case Promise:
-				return createElement(element)
-			case Boolean:
-				break
+				return elementText(element, SharedElementEmpty)
 			default:
-				return elementUnknown(element)
+				return elementUnknown(element, SharedElementEmpty)
```

Comment:
`SharedElementEmpty` seems like a weird choice given the variable name is `index`